src/Altinn.Notifications.Email.Integrations/Consumers/EmailSendingAcceptedConsumer.cs

```diff
@@ -43,9 +48,18 @@ private async Task ConsumeOperation(string message)
 
         if (!succeeded)
         {
+            _logger.LogError("// EmailSendingAcceptedConsumer // ConsumeOperation // Deserialization of message failed. {Message}", message);
+
             return;
         }
 
+        int diff = (int)(DateTime.UtcNow - operationIdentifier.LastStatusCheck).TotalMilliseconds;
```

Comment:
Should we consider using the time service here?

Reply:
yup